core/src/main/java/org/elasticsearch/tasks/TaskManager.java

```
@@ -54,8 +64,28 @@ public Task register(String type, String action, TransportRequest request) {
             if (logger.isTraceEnabled()) {
                 logger.trace("register {} [{}] [{}] [{}]", task.getId(), type, action, task.getDescription());
             }
+            try {
+                task.onRegistered();
+            } catch (Throwable t) {
```

can we make this fail hard?I don't think the logging buys us anything?